src/components/FolderView.tsx

```diff
@@ -36,6 +36,23 @@ export const FolderView = ({
     currentfilter,
   );
 
+  if (filteredCourseContent?.length === 0) {
+    const filterMessages: Record<string, string> = {
```

Comment:
Best way to write typescript is to avoid writing types and letting the compiler decide

```suggestion
    const filterMessages = {
```